utils/types/priority.go

```diff
@@ -108,3 +108,19 @@ func (p *Priority) Scan(value interface{}) error {
 func (p *Priority) Value() (driver.Value, error) {
 	return string(*p), nil
 }
+
+func ScoreToPriority(score float32) Priority {
```

Comment:
Add a comment explaining that it deals with CVSS scores, on a scale from 0 to 10.